.github/workflows/additional.yml

```diff
@@ -18,9 +18,9 @@ jobs:
       - name: Setup Conda Environment
         uses: conda-incubator/setup-miniconda@v3.0.4
         with:
-          miniforge-variant: Mambaforge
```

Comment:
Removing this was also needed, but switching to miniforge instead of mambaforge is fine as they're now identical and mambaforge is actually being sunsetted https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/